editor/editor_themes.cpp

```diff
@@ -508,6 +508,7 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	const Color font_disabled_color = Color(mono_color.r, mono_color.g, mono_color.b, 0.35);
 	const Color font_readonly_color = Color(mono_color.r, mono_color.g, mono_color.b, 0.65);
 	const Color font_placeholder_color = Color(mono_color.r, mono_color.g, mono_color.b, 0.6);
+	const Color font_outline_color = font_color;
```

Comment:
It's better to set it to a transparent color. Most controls are written in a way that checks for outline size being over 0 and outline color having alpha over 0.

Reply:
done.